Tests/L0/AzureRmWebAppDeployment/_suite.ts

```diff
@@ -0,0 +1,327 @@
+/// <reference path="../../../definitions/mocha.d.ts"/>
+/// <reference path="../../../definitions/node.d.ts"/>
+/// <reference path="../../../definitions/shelljs.d.ts"/>
+
+import assert = require('assert');
+import trm = require('../../lib/taskRunner');
+import path = require('path');
+import shell = require ('shelljs');
+
+function setResponseFile(name: string) {
+    process.env['MOCK_RESPONSES'] = path.join(__dirname, name);
+}
+
+describe('AzureRmWebAppDeployment Suite', function() {
+    this.timeout(20000);
+
+    var taskSrcPath = path.join (__dirname, '..', '..', '..', 'Tasks', 'AzureRmWebAppDeployment');
+    var testSrcPath = path.join (__dirname, '..', '..', '..', '..', 'Tests', 'L0', 'AzureRmWebAppDeployment');
+
+    before((done) => {
+        // init here
+
+        if(shell.test ('-d', taskSrcPath)) {
+             
+            // Move mocked AzureRMUtil and MSDeployUtility Libraries to task's test location
+            shell.mv( '-f', path.join (taskSrcPath,'AzureRMUtil.js'), path.join (taskSrcPath,'AzureRMUtil_backup.js'));
+            shell.cp(path.join (testSrcPath, 'AzureRMUtil.js'), path.join (taskSrcPath,'AzureRMUtil.js'));
+
+            shell.mv( '-f', path.join (taskSrcPath,'MSDeployUtility.js'), path.join (taskSrcPath,'MSDeployUtility_backup.js'));
+            shell.cp(path.join (testSrcPath, 'MSDeployUtility.js'), path.join (taskSrcPath,'MSDeployUtility.js'));
+        }
+
+        done();
+    });
+
+    after(function() {
```

Comment:
Is after method execute always even if test breaks ? . If no then  task files are  not getting restored since after method will not execute.